pkg/verifier/aws/entry_point.go

```diff
@@ -76,15 +93,6 @@ func (a *AwsVerifier) ValidateEgress(vei verifier.ValidateEgressInput) *output.O
 		return a.Output.AddError(err) // fatal
 	}
 
-	if instanceReadyErr := a.AwsClient.WaitForEC2InstanceCompletion(vei.Ctx, instanceID); instanceReadyErr != nil {
```

Comment:
Was this removal on purpose? It needs to remain in the code

Reply:
We are replacing this with the NewInstanceRunningWaiter which waits for the instance to be running for upto 5 mins.

Reply:
@anispate is this replacement going to be part of this PR ?

Reply:
Done in #149